src/pages/Projects/Components/ProjectEntry/ProjectEntry.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import {Col} from 'react-bootstrap';
+
+class ProjectEntry extends Component {
+
+  render () {
```

Comment:
Se acostumbra a hacer destructing de las props/state antes del render

`const {entry} = this.props;`

Esto evita tener que repetir una y otra vez `this.props` en el render, que deja mas comprensible el codigo JSX